src/dict/dictionary.cc

```diff
@@ -271,8 +271,6 @@ bool Class::loadIconFromFile( QString const & _filename, bool isFullName )
     if ( !img.isNull() ) {
       // Load successful
 
-      //some icon is very large ,will crash the application.
-      img = img.scaledToWidth( 64 );
```

Comment:
This subsequent region of code can be deleted.

As we previously discussed, just down scaled to 64*64 with this method or another method https://doc.qt.io/qt-6/qimage.html#scaled-1 like `auto result = img.scaled.....`

<img width="500" src="https://github.com/user-attachments/assets/8cbceb7f-5ec7-4b34-b437-32a6dd2b5612">